app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/adapters/recycleradapters/FeedbackAdapter.kt

```diff
@@ -44,28 +50,45 @@ class FeedbackAdapter(
         return 0
     }
 
+    private fun arrangeFeedbacks() {
+        arrangedFeedbackList.clear()
+        feedbackResponse.feedbackList.forEach { item ->
+            if (item.email == PrefManager.getStringSet(Constant.SAVED_EMAIL)?.iterator()?.next()) {
+                arrangedFeedbackList.add(item)
+            }
+        }
+
+        var reverseResponse = feedbackResponse.feedbackList.asReversed()
```

Comment:
Where are you reassigning this? I told you last time to check before submitting PRs for review

Reply:
Updated